test/test_landing_game_group.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+import pygame
+from src.landing_game_group import LandingGameGroup
+
+
+@pytest.fixture
+def group():
+    pygame.init()
+    group = LandingGameGroup()
+    yield group
+    pygame.quit()
+
+
+@pytest.fixture
+def sprites():
+    sprite1 = pygame.sprite.Sprite()
+    sprite2 = pygame.sprite.Sprite()
+    return sprite1, sprite2
+
+
+def test_name_object(group, sprites):
+    sprite1, _ = sprites
+    group.add(sprite1)
+    group.name_object("sprite1", sprite1)
+    assert group.get_object_by_name("sprite1") == sprite1
```

Comment:
`is` instead of `==` would ensure that we do not have an equally looking copy of sprite1 here

Reply:
Ouh, didn't know that